File: MyRecyclerView/app/src/main/java/com/example/recycler/myrecyclerview/view/ViewInterface.java
package com.example.recycler.myrecyclerview.view;

import com.example.recycler.myrecyclerview.data.ListItem;

import java.util.List;

/**
 * Created by kellybentes on 16/01/18.
 */

public interface ViewInterface {

    void startDetailActivity (String dateAndTime, String message, int colorResource);
    void setUpAdapterAndView (List<ListItem> listOfData);
}

File: MyRecyclerView/app/src/main/java/com/example/recycler/myrecyclerview/view/ViewInterface.kt
package com.example.recycler.myrecyclerview.view

import com.example.recycler.myrecyclerview.data.ListItem

/**
 * Created by kellybentes on 16/01/18.
 */

interface ViewInterface {

    fun startDetailActivity(dateAndTime: String, message: String, colorResource: Int)
    fun setUpAdapterAndView(listOfData: List<ListItem>)
}
